backend/tools/push_backend_framework_to_s3.bash

```diff
@@ -4,7 +4,7 @@ set -e
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 FAT_BIN_DIR=${SCRIPT_DIR}/../../build/fat_bin
-CURRENT_VERSION=$(cat ${SCRIPT_DIR}/../../version)
+CURRENT_VERSION=0.14.0
```

Comment:
This shouldn't be hard-coded either, again it should come from the git tag.

Reply:
Right, but that's not part of this PR. Should I go back to `$(cat ${SCRIPT_DIR}/../../version)` in the meantime? Because that won't work. At least with this we have the right version for the record (in the `v0.14.0` tag, which we don't have for `v0.13.0` already).

Reply:
Use this:
https://github.com/Dronecode/DronecodeSDK/blob/dc9cfe0fd3ff6b4d17afced662e8f0b588043525/create_packages.sh#L5-L8